src/common/theme/index.ts

```diff
@@ -0,0 +1,2 @@
+import { theme } from './theme'
+export default theme
```

Comment:
Add in settings for your eslint adding extra space. Almost all files have to have an empty row.
![image](https://user-images.githubusercontent.com/63092716/144588970-6fb2e39e-06c0-451e-9b31-8fa7040bc0bc.png)